packages/headless/src/composables/navigationMenu/createMenu.testing.ts

```diff
@@ -0,0 +1,48 @@
+import { expect } from "@playwright/experimental-ct-vue";
+import type { Locator } from "@playwright/test";
+
+export type NavigationMenuTestingOptions = {
+  /**
+   * Locator for the navigation landmark.
+   */
+  nav: Locator;
+  /**
+   * Locator for the button elements.
+   */
+  buttons: Locator;
+};
+
+/**
+ * Playwright utility for executing accessibility testing for a navigation menu.
+ * Will check aria attributes and keyboard shortcuts as defined in https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/
+ */
+export const navigationTesting = async ({ nav, buttons }: NavigationMenuTestingOptions) => {
```

Comment:
Seems like this is not used in any onyx component 🤔

Reply:
Will do in follow-up